apps/mobile-wallet/src/App.tsx

```diff
@@ -80,12 +80,26 @@ const App = () => {
     []
   )
 
+  const toastConfig: BaseToastProps = {
```

Comment:
Nit: This is static and could be taken out of the component. I'll do that while I'm tweaking the layout a bit.

Reply:
(so it's fine for now!)